review-cask-pr/review.rb

```diff
@@ -177,8 +177,11 @@ def review_diff(pr, author_is_member: false)
 
       puts message
 
-      puts "::set-output name=event::#{event}"
-      puts "::set-output name=message::#{GitHub::Actions.escape(message)}"
+      github_output = ENV.fetch("GITHUB_OUTPUT") { raise "GITHUB_OUTPUT is not defined" }
+      File.open(github_output, "a") do |f|
+        f.puts("event=#{event}")
```

Comment:
Same here. This is way more Homebrew API Ruby than we should be having in this repo.